SubmittedAssignments/view-invoice-format2/index.html

```
@@ -0,0 +1,102 @@
+              "customerDetailsClass":"customer-details",
+              "customerHeaderClass":"customer-header",
+              "billingDetailsClass":"billing-details",
+              "shippingDetailsClass":"shipping-details",
+              "tableClass": "content-table",
+              "tableHeaderClass": "content-header",
+              "totalClass":"total",
+              "amountDetailsClass":"amount-details",
+              "footerClass": "invoice-footer"
+
+            }'
+             data ='{
```

HTMLHint (1.5.0)

[error] The value of attribute [ data ] must be in double quotes.

(attr-value-double-quotes)